airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/utils/client_container_runner.py

```diff
@@ -54,7 +55,7 @@ async def _run_with_config(container: dagger.Container, command: List[str], conf
 
 
 async def _run(container: dagger.Container, command: List[str]) -> dagger.Container:
-    return await container.with_exec(command, skip_entrypoint=True)
+    return await (container.with_env_variable("CACHEBUSTER", str(uuid.uuid4())).with_exec(command, skip_entrypoint=True))
```

Comment:
If we don't add this cache-busting layer, the `with_exec` layer itself can be cached. This became apparent when the container setup auto-fixture didn't actually run like I thought it would (because the run was identical to another, earlier test case)